components/centraldashboard/Dockerfile

```diff
@@ -1,20 +1,7 @@
-FROM golang:1.10 as builder
-
-# Download and install the latest release of dep
-ADD https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64 /usr/bin/dep
-RUN chmod +x /usr/bin/dep
-
-# Copy the code from the host and compile it
-WORKDIR $GOPATH/src/github.com/kubeflow/kubeflow/components/centraldashboard
-COPY Gopkg.toml Gopkg.lock ./
-RUN dep ensure --vendor-only
-COPY . ./
-RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix nocgo -o centraldashboard .
-
-FROM alpine
+FROM ARCH/alpine
 RUN apk --no-cache add ca-certificates
```

Comment:
Need leverage [multiarch project](https://github.com/multiarch) project to make this work, so we need three base images for amd64,ppc64le,s390x.